packages/app/src/app/overmind/namespaces/live/liveMessageOperators.ts

```diff
@@ -196,6 +198,8 @@ export const onUserLeft = (
     return;
   }
 
+  effects.analytics.track('Editor - Left the live session');
```

Comment:
This will be called when _any_ user leaves, so not only the current user. If that's intended, then it might be better to name that in the track name (e.g. `Editor - Someone left the live session`)